packages/x-charts/src/ChartsVoronoiHandler/ChartsVoronoiHandler.tsx

```diff
@@ -107,18 +107,18 @@ function ChartsVoronoiHandler(props: ChartsVoronoiHandlerProps) {
   }, [defaultXAxisId, defaultYAxisId, series, seriesOrder, xAxis, yAxis, drawingArea]);
 
   React.useEffect(() => {
-    const element = svgRef.current;
-    if (element === null) {
-      return undefined;
-    }
-
     function getClosestPoint(
       event: MouseEvent,
     ):
       | { seriesId: SeriesId; dataIndex: number }
       | 'outside-chart'
       | 'outside-voronoi-max-radius'
       | 'no-point-found' {
+      const element = svgRef.current;
+      if (element === null) {
+        return 'no-point-found';
+      }
+
```

Comment:
I don't get that point. If their is not svg element to attach events to it why this function should return `'no-point-found'`?

Reply:
It don't know which one it should return, `'outside-chart'` maybe? The issue here is that `getSVGPoint` requires the element be present

Reply:
Yes, that's why the check for `element === null` was initially at the beginning. If the element is `null`, we will not attach events to it so we can do early return.

If your concern is about the risk of mutation, notice it's the SVG ref. The moment it's null, it means we have no chart to display, so we have bigger issue in that case :)

Reply:
got it, I moved it cause it was giving me errors, but now it doesn't anymore, so I guess I can revert everything...